src/main/scala/ScalatraBootstrap.scala

```diff
@@ -0,0 +1,22 @@
+import javax.servlet.ServletContext
+
+import no.ndla.crowdingateway.{ComponentRegistry, CrowdinGatewayProperties}
+import org.scalatra.LifeCycle
+
+/*
+ * Part of NDLA crowdin_gateway.
+ * Copyright (C) 2017 NDLA
+ *
+ * See LICENSE
+ */
+
+
+class ScalatraBootstrap extends LifeCycle {
+
+  override def init(context: ServletContext) {
+    context.mount(ComponentRegistry.crowdinController, CrowdinGatewayProperties.CrowdinControllerPath, "crowdin")
+    context.mount(ComponentRegistry.resourcesApp, "/api-docs")
```

Comment:
Det bør kanskje lages en entry i `CrowdinGatewayProperties` for api-docs også? siden det er slik for de andre controllerene